Makefile

```diff
@@ -119,21 +115,11 @@ srpm: dist-gzip $(PACKAGE_NAME).spec
 	  --define "_srcrpmdir `pwd`" \
 	  $(PACKAGE_NAME).spec
 
-rpm: dist-gzip $(PACKAGE_NAME).spec
-	mkdir -p "`pwd`/output"
-	mkdir -p "`pwd`/rpmbuild"
-	rpmbuild -bb \
-	  --define "_sourcedir `pwd`" \
-	  --define "_specdir `pwd`" \
-	  --define "_builddir `pwd`/rpmbuild" \
-	  --define "_srcrpmdir `pwd`" \
-	  --define "_rpmdir `pwd`/output" \
-	  --define "_buildrootdir `pwd`/build" \
-	  $(MACRO_RHEL) \
-	  $(PACKAGE_NAME).spec
-	find `pwd`/output -name '*.rpm' -printf '%f\n' -exec mv {} . \;
-	rm -r "`pwd`/rpmbuild"
-	rm -r "`pwd`/output" "`pwd`/build"
+# this is a noarch build, so local rpm build works fine mostly; the exception
+# is continuous-atomic, which does not get along with rpms build on Fedora ≥ 31
+rpm: srpm
+	srpm=`ls *.src.rpm | head -n1` && \
+	[ "$$TEST_OS" = continuous-atomic ] && test/rpmbuild-vm "$$srpm" centos-7 || test/rpmbuild-local "$$srpm"
```

Comment:
If `centos-7` image does not exist this rpmbuild-vm command will fail and this will end up in local rpm-build which had the known-problem with continuous atomic.

Reply:
Thanks for spotting! Fixed.